### ISSUE TITLE
java.lang.IllegalStateException: Bytecode creation for the 'SSLSocket' class is not possible due to unmet preconditions: Caused by: java.lang.IllegalStateException: Cannot find max value

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/jeo-maven-plugin#793]
I've added this dependency to `src/it/spring-fat/pom.xml`:

```xml
<dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-web</artifactId>
</dependency>
```

I used plugin of `@project.version@` version:

```xml
<groupId>org.eolang</groupId>
<artifactId>jeo-maven-plugin</artifactId>
<version>@project.version@</version>
```

then, I run:

```bash
mvn clean integration-test -Dinvoker.test=spring-fat -DskipTests
```

And got an error:

```text
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:assemble (eo-to-bytecode) on project jeo-spring-fat-it: Execution eo-to-bytecode of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:assemble failed: java.lang.IllegalStateException: Can't transform XMIR to bytecode from the '/Users/aliaksei/code/open/jeo-maven-plugin/target/it/spring-fat/target/generated-sources/jeo-xmir/org/apache/tomcat/jni/SSLSocket.xmir' source
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution eo-to-bytecode of goal org.eolang:jeo-maven-plugin:1.0-SNAPSHOT:assemble failed: java.lang.IllegalStateException: Can't transform XMIR to bytecode from the '/Users/aliaksei/code/open/jeo-maven-plugin/target/it/spring-fat/target/generated-sources/jeo-xmir/org/apache/tomcat/jni/SSLSocket.xmir' source
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:133)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: java.lang.IllegalStateException: java.lang.IllegalStateException: Can't transform XMIR to bytecode from the '/Users/aliaksei/code/open/jeo-maven-plugin/target/it/spring-fat/target/generated-sources/jeo-xmir/org/apache/tomcat/jni/SSLSocket.xmir' source
    at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance (DirectConstructorHandleAccessor.java:62)
    at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:502)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:486)
    at java.util.concurrent.ForkJoinTask.getThrowableException (ForkJoinTask.java:540)
    at java.util.concurrent.ForkJoinTask.reportException (ForkJoinTask.java:567)
    at java.util.concurrent.ForkJoinTask.invoke (ForkJoinTask.java:670)
    at java.util.stream.ForEachOps$ForEachOp.evaluateParallel (ForEachOps.java:160)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel (ForEachOps.java:174)
    at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:233)
    at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:596)
    at org.eolang.jeo.Assembler.assemble (Assembler.java:77)
    at org.eolang.jeo.AssembleMojo.execute (AssembleMojo.java:115)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
Caused by: java.lang.IllegalStateException: Can't transform XMIR to bytecode from the '/Users/aliaksei/code/open/jeo-maven-plugin/target/it/spring-fat/target/generated-sources/jeo-xmir/org/apache/tomcat/jni/SSLSocket.xmir' source
    at org.eolang.jeo.representation.XmirRepresentation.toBytecode (XmirRepresentation.java:121)
    at org.eolang.jeo.Assemble.apply (Assemble.java:59)
    at org.eolang.jeo.LoggedTranslation.apply (LoggedTranslation.java:84)
    at org.eolang.jeo.BachedTranslator.translate (BachedTranslator.java:71)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:179)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.Spliterators$ArraySpliterator.forEachRemaining (Spliterators.java:1024)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
Caused by: java.lang.IllegalStateException: Bytecode creation for the 'SSLSocket' class is not possible due to unmet preconditions. To reproduce the problem, you can write the following test: 
AllLabels labels = new AllLabels();
new BytecodeClass("SSLSocket")
.withMethod("<init>", "()V", 1)
//maxs BytecodeMaxs(stack=1, locals=1).label("d8597399-9014-41fa-b4fb-f0fa5acc1aeb")
.opcode(Opcodes.aload, 0)
.opcode(Opcodes.invokespecial, "java/lang/Object", "<init>", "()V", false)
.opcode(Opcodes.return)
.label("ea1f15b1-60af-4b99-9c9b-0356134a6b4f")
.up()
.withMethod("attach", "(JJ)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("handshake", "(J)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("renegotiate", "(J)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("setVerify", "(JII)V", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoB", "(JI)[B", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoS", "(JI)Ljava/lang/String;", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoI", "(JI)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getALPN", "(J[B)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.bytecode();

    at org.eolang.jeo.representation.bytecode.BytecodeClass.writeTo (BytecodeClass.java:365)
    at org.eolang.jeo.representation.bytecode.BytecodeProgram.bytecode (BytecodeProgram.java:107)
    at org.eolang.jeo.representation.XmirRepresentation.toBytecode (XmirRepresentation.java:113)
    at org.eolang.jeo.Assemble.apply (Assemble.java:59)
    at org.eolang.jeo.LoggedTranslation.apply (LoggedTranslation.java:84)
    at org.eolang.jeo.BachedTranslator.translate (BachedTranslator.java:71)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:179)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.Spliterators$ArraySpliterator.forEachRemaining (Spliterators.java:1024)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
Caused by: java.lang.IllegalStateException: Failed to generate bytecode method BytecodeMethodProperties(access=265, name=attach, descr=(JJ)I, signature=, parameters=BytecodeMethodParameters(params=[BytecodeMethodParameter(index=0, type=J, annotations=BytecodeAnnotations(all=[])), BytecodeMethodParameter(index=1, type=J, annotations=BytecodeAnnotations(all=[]))]), exceptions=[java/lang/Exception]) due to unexpected exception
    at org.eolang.jeo.representation.bytecode.BytecodeMethod.write (BytecodeMethod.java:355)
    at org.eolang.jeo.representation.bytecode.BytecodeClass.lambda$writeTo$2 (BytecodeClass.java:355)
    at java.util.ArrayList.forEach (ArrayList.java:1596)
    at org.eolang.jeo.representation.bytecode.BytecodeClass.writeTo (BytecodeClass.java:355)
    at org.eolang.jeo.representation.bytecode.BytecodeProgram.bytecode (BytecodeProgram.java:107)
    at org.eolang.jeo.representation.XmirRepresentation.toBytecode (XmirRepresentation.java:113)
    at org.eolang.jeo.Assemble.apply (Assemble.java:59)
    at org.eolang.jeo.LoggedTranslation.apply (LoggedTranslation.java:84)
    at org.eolang.jeo.BachedTranslator.translate (BachedTranslator.java:71)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:179)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.Spliterators$ArraySpliterator.forEachRemaining (Spliterators.java:1024)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
Caused by: java.lang.IllegalStateException: Cannot find max value
    at org.eolang.jeo.representation.bytecode.InstructionsFlow.lambda$max$4 (InstructionsFlow.java:134)
    at java.util.Optional.orElseThrow (Optional.java:403)
    at org.eolang.jeo.representation.bytecode.InstructionsFlow.max (InstructionsFlow.java:134)
    at org.eolang.jeo.representation.bytecode.MaxStack.value (MaxStack.java:66)
    at org.eolang.jeo.representation.bytecode.BytecodeMethod.computeStack (BytecodeMethod.java:434)
    at org.eolang.jeo.representation.bytecode.BytecodeMethod.write (BytecodeMethod.java:324)
    at org.eolang.jeo.representation.bytecode.BytecodeClass.lambda$writeTo$2 (BytecodeClass.java:355)
    at java.util.ArrayList.forEach (ArrayList.java:1596)
    at org.eolang.jeo.representation.bytecode.BytecodeClass.writeTo (BytecodeClass.java:355)
    at org.eolang.jeo.representation.bytecode.BytecodeProgram.bytecode (BytecodeProgram.java:107)
    at org.eolang.jeo.representation.XmirRepresentation.toBytecode (XmirRepresentation.java:113)
    at org.eolang.jeo.Assemble.apply (Assemble.java:59)
    at org.eolang.jeo.LoggedTranslation.apply (LoggedTranslation.java:84)
    at org.eolang.jeo.BachedTranslator.translate (BachedTranslator.java:71)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:179)
    at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:197)
    at java.util.Spliterators$ArraySpliterator.forEachRemaining (Spliterators.java:1024)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:509)
    at java.util.stream.ForEachOps$ForEachTask.compute (ForEachOps.java:291)
    at java.util.concurrent.CountedCompleter.exec (CountedCompleter.java:754)
    at java.util.concurrent.ForkJoinTask.doExec (ForkJoinTask.java:387)
    at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec (ForkJoinPool.java:1312)
    at java.util.concurrent.ForkJoinPool.scan (ForkJoinPool.java:1843)
    at java.util.concurrent.ForkJoinPool.runWorker (ForkJoinPool.java:1808)
    at java.util.concurrent.ForkJoinWorkerThread.run (ForkJoinWorkerThread.java:188)
```

Context: after #787 fix I was able to pass `jeo-maven-plugin:disassemble` goal, however, on `jeo-maven-plugin:assemble` goal it failed

[build.log](https://github.com/user-attachments/files/17489671/build.log)

**Comment by h1alexbel** [comment]
@volodya-lombrozo take a look, please

**Comment by h1alexbel** [comment]
@volodya-lombrozo I will open a PR that reproduces this:

```text
AllLabels labels = new AllLabels();
new BytecodeClass("SSLSocket")
.withMethod("<init>", "()V", 1)
//maxs BytecodeMaxs(stack=1, locals=1).label("d8597399-9014-41fa-b4fb-f0fa5acc1aeb")
.opcode(Opcodes.aload, 0)
.opcode(Opcodes.invokespecial, "java/lang/Object", "<init>", "()V", false)
.opcode(Opcodes.return)
.label("ea1f15b1-60af-4b99-9c9b-0356134a6b4f")
.up()
.withMethod("attach", "(JJ)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("handshake", "(J)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("renegotiate", "(J)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("setVerify", "(JII)V", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoB", "(JI)[B", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoS", "(JI)Ljava/lang/String;", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getInfoI", "(JI)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.withMethod("getALPN", "(J[B)I", 265)
//maxs BytecodeMaxs(stack=0, locals=0).up()
.bytecode();
```

**Comment by volodya-lombrozo** [comment]
@h1alexbel It seems that this issue is a duplicate of https://github.com/objectionary/jeo-maven-plugin/issues/790. What do you think? 
Most probably, the problem will be solved here: https://github.com/objectionary/jeo-maven-plugin/pull/792

**Comment by h1alexbel** [comment]
@volodya-lombrozo now it works, can be closed
